File: MarsalaApi/src/WebApplication1/APP/Models/dailyMenu.ts
export interface IDailyMenu {
    header: string;
    salad: string[];
    soup: string[];
    mainCourse: string[];
    drink: string[]; 
}



export class DailyMenu implements IDailyMenu {
    header: string;
    salad: string[];
    soup: string[];
    mainCourse: string[];
    drink: string[];


    constructor(header: string, salad: string[], soup: string[], mainCourse: string[], drink: string[]) {
        this.header = header;
        this.salad = salad;
        this.soup = soup;
        this.mainCourse = mainCourse;
        this.drink = drink;
    }

}

File: MarsalaApi/src/Common/Model/DailyMenu.cs
using System.Collections.Generic;

namespace Common.Model
{
	public class DailyMenu
	{
		public string Header { get; set; }
		public ICollection<string> Salad { get; set; }
		public ICollection<string> Soup { get; set; }
		public ICollection<string> MainCourse { get; set; }
		public ICollection<string> Drink { get; set; }
	}
}
